kofta/tsconfig.json

```diff
@@ -1,20 +1,21 @@
 {
-	"compilerOptions": {
-		"target": "es5",
-		"lib": ["dom", "dom.iterable", "esnext"],
-		"allowJs": true,
-		"skipLibCheck": true,
-		"esModuleInterop": true,
-		"allowSyntheticDefaultImports": true,
-		"strict": true,
-		"forceConsistentCasingInFileNames": true,
-		"noFallthroughCasesInSwitch": true,
-		"module": "esnext",
-		"moduleResolution": "node",
-		"resolveJsonModule": true,
-		"isolatedModules": true,
-		"noEmit": true,
-		"jsx": "react"
-	},
-	"include": ["src"]
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noFallthroughCasesInSwitch": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react",
+    "noImplicitAny": false
```

Comment:
I don't personally get any of the errors you mentioned in your issue, so i don't believe this is a necessary change.
Something might be wrong with your fork.

Reply:
Or it might even be the fact that you are explicitly stating `noImplicitAny: false`

Reply:
I think that the explicit statement of that is the resolution to the bug @samarmohan was getting, but I could have misunderstood.

Reply:
@amitojsingh366, I have only added `noImplicitAny: false` Because in the code, there are a lot of implicit anys. That makes editors throw errors and we obviously don’t like errors.

Reply:
@samarmohan but the thing is this isnt happening for anyone else... this is happening only for you so i don't think this is a valid pull

Reply:
Which IDE or editor are you using?

this is in VSCode, I will test in in IntelliJ based

Reply:
im using vscode...

Reply:
Also did `noImplicitAny: false` fix your issue or not?

Reply:
it did

Reply:
oh ok i thought it didn't. my bad, sorry for the misunderstanding